website/docs/guides/mangabox.md

```diff
@@ -0,0 +1,32 @@
+# Read with MangaBox
+
+[MangaBox](https://zpaolo11x.github.io/mangabox/) is a streamlined, multiplatform Komga client. It's free and open-source, available for Windows, macOS[(1)](#note1), Linux, iOS/ipadOS[(2)](#note2) and Android[(3)](#note3). It is based on web technologies so you can also deploy it as a web client or PWA.
+
+<img src="/assets/media/guides/mangabox/mangabox.png" style={{maxHeight: '500px'}}/>
+
+## Features
+
+- Connect easily to your Komga server from the onboarding screen
+- Light and dark themes with custom color highlights
+- Touch and gesture support for tablets and phones
+- Beautiful two-pages reading with paper-style visuals, page-turn animations and thumbnail previews
+- Sync reading progress with the server
+- Download books for offline reading
+- Quick filter libraries by language, read status, and direction
+- Global search for series and volumes
+
+## Track read progress
+
+Mangabox can sync your reading progress with the server. When you read a book offline, read progress is synced back once the server is reachable again.
+
+## Useful links
+
+- [MangaBox Website](https://zpaolo11x.github.io/mangabox): The official MangaBox website, with the links to the latest builds, features list and a preview of the reader UI.
+- [Github Repository](https://github.com/zpaolo11x/mangabox): The main repository for issue tracking, old releases etc.
+- [Discord Server](https://discord.com/invite/PEcjGMJR5C): beside being hosted in the Komga discord server, you can also join MangaBox server for assistance and discussions.
+
+## Notes
+
+1. <a id="note1"></a>macOS binary is vurrently unsigned, you can visit MangaBox website for instructions on how to self-sign it.
```

Comment:
apart from the typ on "currently", i don't think this note is necessary. It should be covered on the Mangabox website, not here.